writer/standard/writer.go

```diff
@@ -193,14 +193,15 @@ func draw(mat qrcode.Matrix, opt *outputImageOptions) image.Image {
 		if !validLogoImage(w, h, logoWidth, logoHeight, opt.logoSizeMultiplier) {
 			log.Printf("w=%d, h=%d, logoW=%d, logoH=%d, logo is over than 1/%d of QRCode \n",
 				w, h, logoWidth, logoHeight, opt.logoSizeMultiplier)
-			goto done
+
+			return dc.Image()
```

Comment:
I couldn't agree that goto is unsafe here, replacing goto by directly return is not necessary

Reply:
It’s one less goto to have to check for security issues. Usually they are dangerous during for loops as you can easily end up with unbounded loops or other issues especially if you aren’t fuzzing. 

In this case it’s simple but I think just adding the return here or switching this to an if else on the ‘dc.DrawImage’ is an easy way to avoid it and thus make security reviews for new users have less red flags to check.